### PR TITLE
Add kernel info and timezone info

### PR DESCRIPTION
**Pull request by vsibirsk**
Add kernel (release, version, type) and timezone (name, offset) info.

### CODE REVIEW

**Comment by ILpinto** [comment]
@lukas-bednar please approve

File: rrmngmnt/operatingsystem.py [hunk 1]
```diff
@@ -114,6 +116,58 @@ def distribution(self):
...
+                    version='#1 SMP Fri Aug 16 19:31:40 UTC 2019',
+                    type'='x86_64'
+                    )
+        """
+        values = ["release", "version", "type"]
+        cmd = ["uname", "-r", ";", "uname", "-v", ";", "uname", "-m"]
+        out = self._exec_command(
+            cmd=cmd, err_msg="Failed to obta kernel info"
+        )
+        Kernel = namedtuple('Kernel', values)
```
Comment (myakove) [inline rrmngmnt/operatingsystem.py:138]: kernel, should be all lowercase

File: rrmngmnt/operatingsystem.py [hunk 2]
```diff
@@ -114,6 +116,58 @@ def distribution(self):
...
+
+        Examples:
+             kernel(name='IDT', offset='+0300')
+        """
+        values = ["name", "offset"]
+        cmd = ["date", "+%Z\\", "%z"]
+        out = self._exec_command(
+            cmd=cmd, err_msg="Failed to obtain timezone info"
+        )
+        Timezone = namedtuple('Timezone', values)
```
Comment (myakove) [inline rrmngmnt/operatingsystem.py:162]: ditto

File: rrmngmnt/operatingsystem.py (hunk 1 above)
Comment (vsibirsk) [reply rrmngmnt/operatingsystem.py:138]: made it from capital just to be aligned with already existing code (see Distribution in get_distribution method)

Comment (myakove) [reply rrmngmnt/operatingsystem.py:138]: This is wrong, but I guess @lukas-bednar should decide.

Comment (lukas-bednar) [reply rrmngmnt/operatingsystem.py:138]: I believe this is alright. `namedtuple` returns a class, and then you create instance of that class. And classes in python should be Uppercased.

File: tests/test_os.py [hunk 3]
```diff
@@ -95,6 +110,8 @@ class TestOperatingSystemNegative(object):
         ),
         'python -c "import platform;print(\',\'.join('
         'platform.linux_distribution()))"': (1, '', 'SyntaxError'),
+        'uname -r ; uname -v ; uname -m': (1, '', 'SyntaxError'),
```
Comment (lukas-bednar) [inline tests/test_os.py:113]: I don't think that `uname` or `date` command can raise any `SyntaxError` , so I believe that these tests are redundant here.
You can either remove them, or think about something more likely happening for these two commands.

**Comment by ILpinto** [comment]
LGTM

**Pull request merged**